component/index.js

```diff
@@ -25,4 +32,15 @@ const showSnackbar = () => {
 }
 
 snackbarShowBtn.addEventListener("click", showSnackbar);
-snackbarHideBtn.addEventListener("click", hideSnackbar);
\ No newline at end of file
+snackbarHideBtn.addEventListener("click", hideSnackbar);
+
+
+// ------ script for Modal --------- 
+showModal.addEventListener("click",() => {
+    modalContainer.style.display = "block";
+})
+
+modalClose.addEventListener("click",()=>{
+    console.log("clicked");
```

Comment:
I guess this console is for testing purpose 
so we can remove this once it is done

Reply:
Yes, I forgot to remove. Thanks for pointing out.